Watership_Down_Exercise/BunnyCreator.cs

```diff
@@ -5,41 +5,42 @@
 using System.Threading.Tasks;
 
 
-namespace Watership_Down_Exercise
-{
-    static class BunnyCreator
-    {
-        private static readonly Random random = new Random();
-        public static Bunny CreateBunny()
-        {
-            Bunny bunny;
-            Bunny.Color randomColor;
-            //Get a random number between 1 and 4
-            int randomNumber = random.Next(5);
-
-            //Generate a random color
-            switch (randomNumber)
-            {
-                case 1:
-                    randomColor = Bunny.Color.Black;
-                    break;
-                case 2:
-                    randomColor = Bunny.Color.Brown;
-                    break;
-                case 3:
-                    randomColor = Bunny.Color.Spotted;
-                    break;
-                case 4:
-                default:
-                    randomColor = Bunny.Color.White;
-                    break;
-            }
-            //Create the bunny with the random color
-            bunny = new Bunny(randomColor);
-
-            //[Male/Female] [Black/Brown/Spotted/White] Bunny [Name] Was born!
-            Console.WriteLine("{0} {1} Bunny {2} Was born!", bunny.getSex(), bunny.getColor(), bunny.GetName());
-            return new Bunny(randomColor);
-        }
-    }
-}
+ namespace Watership_Down_Exercise
+ {
+     //The BunnyCreator class is used to create bunnies
+     static class BunnyCreator
+     {
+         private static readonly Random random = new Random();
+         public static Bunny CreateBunny()
+         {
+             Bunny bunny;
+             Bunny.Color randomColor;
+             //Get a random number between 1 and 4
+             int randomNumber = random.Next(5);
+ 
+             //Generate a random color
+             switch (randomNumber)
+             {
+                 case 1:
+                     randomColor = Bunny.Color.Black;
+                     break;
+                 case 2:
+                     randomColor = Bunny.Color.Brown;
+                     break;
+                 case 3:
+                     randomColor = Bunny.Color.Spotted;
+                     break;
+                 case 4:
+                 default:
+                     randomColor = Bunny.Color.White;
+                     break;
+             }
+             //Create the bunny with the random color
+             bunny = new Bunny(randomColor);
+ 
+             //[Male/Female] [Black/Brown/Spotted/White] Bunny [Name] Was born!
+             Console.WriteLine("{0} {1} Bunny {2} Was born!", bunny.getSex(), bunny.getColor(), bunny.GetName());
+             return new Bunny(randomColor);
```

Comment:
Why are you returning a new bunny instead of the 'bunny' you have already created?